cts/scheduler/scores/colocation-priority-group.scores

```diff
@@ -0,0 +1,19 @@
+
+pcmk__group_assign: group1 allocation score on node1: 0
+pcmk__group_assign: group1 allocation score on node2: 0
+pcmk__group_assign: member1a allocation score on node1: 0
+pcmk__group_assign: member1a allocation score on node2: 0
+pcmk__group_assign: member1b allocation score on node1: 0
+pcmk__group_assign: member1b allocation score on node2: 0
+pcmk__primitive_assign: Fencing allocation score on node1: 0
+pcmk__primitive_assign: Fencing allocation score on node2: 0
+pcmk__primitive_assign: member1a allocation score on node1: 0
+pcmk__primitive_assign: member1a allocation score on node2: -4000
```

Comment:
Yeah, in pcmk__apply_coloc_to_weights(), if the primary isn't running, the negative of the score is applied to all the dependent's allowed nodes. Since it's the same for all nodes, presumably it would matter only if INFINITY. (Anti-colocations have already been filtered out by that point.)

Reply:
Ah okay, so it sounds like the 0 is from 4000 + (-4000) then too. (crm_simulate -VVVVVV probably would have told me that)